src/components/contactForm/ContactForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+
+export const ContactForm = ({addContact}) => {
+    const [name, setName] = useState('');
+    const [phone, setPhone] = useState('');
+    const [email, setEmail] = useState('');
+    const [phoneValid, setPhoneValid] = useState(false);
+    const [emailValid, setEmailValid] = useState(false);
+    const [submitValid, setSubmitValid] = useState(false);
+
+    const handleSubmit = e => {
+      e.preventDefault();
+      addContact(name, phone, email);
+    };
+
+    useEffect(() => {
+      const phoneRegex = /[1-9][0-9]{2}-[1-9][0-9]{2}-[0-9]{4}/;
```

Comment:
I'm not sure if these regex definitions belong in their `useEffect` method body or at the top level of the component. You'll have to check with John on that one.

Reply:
Additionally: make sure the regex is flexible to international numbers. Over half of our users live outside the US

Reply:
Let's avoid using our own regexes and leverage HTML's input validation.

What we'll need to do:

1. Remove the xValid state members
2. Remove these effects
3. Add the validation attributes to `input`

Reply:
Done.